modules/ROOT/partials/integrations/vue-quick-start.adoc

```diff
@@ -36,32 +36,21 @@ This procedure requires https://nodejs.org/[Node.js (and npm)].
 
 == Procedure
 
-. On a command line or command prompt, install the https://cli.vuejs.org/#getting-started[Vue CLI Tool] package.
+. Create a new Vue project named `tinymce-vue-demo` using the https://github.com/vuejs/create-vue[Create Vue Tool].
+* From a command line or command prompt create a Vue.js 3.x project: `+tinymce-vue-demo+`.
 +
 [source,sh]
 ----
-npm install -g @vue/cli
+npm create vue@3
 ----
-. Create a new Vue.js project named `+tinymce-vue-demo+`.
-* To use the interactive prompt, run:
+* If you need to create a Vue.js 2.x projects instead:
 +
 [source,sh]
 ----
-vue create tinymce-vue-demo
-----
-* To skip the interactive prompt:
-** For Vue.js 3.x users:
-+
-[source,sh]
-----
-vue create --inlinePreset '{ "vueVersion": "3", "plugins": {} }' tinymce-vue-demo
-----
-** For Vue.js 2.x users:
-+
-[source,sh]
-----
-vue create --inlinePreset '{ "vueVersion": "2", "plugins": {} }' tinymce-vue-demo
+npm create vue@2
 ----
+NOTE: Vue 2 will reach End of Life by the end of 2023 according to https://vuejs.org/about/faq.html#what-s-the-difference-between-vue-2-and-vue-3[Vue FAQ].
```

Comment:
```suggestion

NOTE: As per the https://vuejs.org/about/faq.html#what-s-the-difference-between-vue-2-and-vue-3[Vue FAQ], _Vue 2 will reach End of Life by the end of 2023_.

```

Added white space above and below the NOTE admonition to move said admonition out of the bulleted list.

Also edited the NOTE text to make it clear we are quoting the Vue FAQ, not paraphrasing or interpreting it.

Reply:
Looks good! Thanks